README.md

```diff
@@ -1,2 +1,168 @@
+
+# Checking the Latest Responses
+
+Execute `node scripts/update-dual-license-responses.js` to update [`dual-license-responses.json`](data/dual-license-responses.json) with the latest comments on the relevant GitHub issues. This script has been tested with node v14.16.0. In order to run this script, you must have the [GitHub CLI](https://cli.github.com/) installed and configured (the GitHub CLI handled pagination of the search results). 
+
+Running that script will download all the comments on each issue, allow the user to process each new comment individually (previously processed comments are skipped), and update the [`dual-license-responses.json`](data/dual-license-responses.json) file appropriately based on the user's input.
+
+This script will automatically run [`summarize-dual-license-responses.js`](scripts/summarize-dual-license-responses.js), which prints out some current status about the current status of the requests for consent and updates the [`status.md`](status.md) file.
+
+# How Was `dual-license-responses.json` Initialized?
+
+The [`dual-license-responses.json`](dual-license-responses.json) json file was initialized with:
+1. All the GitHub users who have contributed to Gutenberg; and 
+2. All of the email addresses that authored a commit on Gutenberg's `trunk` branch but was not associated with any GitHub account. We want to identify the GitHub users behind these commits and ask those GitHub users to consent to dual-license their past contributions.
+
+## How Gutenberg’s Contributors Were Identified
+
+This data was collected on April 16, 2021
+
+The easiest way to execute the GraphQL queries listed below is by using the [GitHub CLI](https://cli.github.com/) to execute a command along these lines `gh api graphql --paginate -f query='query(...'`, which will handle authentication and pagination for you.
+
+### Contributors With GitHub Logins
+
+#### All Merged and Open PRs
+
+Saved as [2021-04-16_merged-or-open-prs.json](data/2021-04-16_merged-or-open-prs.json)
+
+<details>
+  <summary>GraphQL query</summary>
+  
+```
+query($endCursor: String) {
+  repository(owner: "WordPress", name: "gutenberg") {
+    pullRequests(states:[MERGED, OPEN] first: 100, after: $endCursor) {
+      totalCount
+      pageInfo {
+        endCursor
+        hasNextPage
+      }
+      nodes {
+        state
+        number
+        author {
+          login
+        }
+        mergedBy {
+          login
+        }
+      }
+    }
+  }
+}'
+```
+
+Note that this query returns _all_ PRs through the date the script is run.
+</details>
+
+Although this query captures logins of both the "author" and the person that merged each PR, the data shows that there is no one who has merged prs who has not also authored at least one PR. In other words, including logins from the "mergedBy" field does not add any additional contributors beyond those captured by the "author" field.
```

Comment:
```suggestion
Although this query captures logins of both the "author" and the person that merged each PR, the data shows that there is no one who has merged a PR who has not also authored at least one PR. In other words, including logins from the "mergedBy" field does not add any additional contributors beyond those captured by the "author" field.
```
This clears an ambiguity I see where it wasn't clear if it meant "people who have merged one or more PRs" or "people who have one or more merged PRs".